opendbc/dbc/generator/hyundai/hyundai_canfd.dbc

```diff
@@ -46,12 +46,15 @@ BO_ 160 WHEEL_SPEEDS: 24 XXX
  SG_ WHL_SpdRLVal : 96|14@1+ (0.03125,0) [0|0] "km^h" XXX
  SG_ WHL_SpdRRVal : 112|14@1+ (0.03125,0) [0|0] "km^h" XXX
 
-BO_ 203 LFA_ALT: 24 ADRV
- SG_ CHECKSUM : 0|16@1+ (1,0) [0|65535] "" XXX
- SG_ COUNTER : 16|8@1+ (1,0) [0|255] "" XXX
- SG_ LKAS_ANGLE_ACTIVE : 29|2@0+ (1,0) [0|1] "" XXX
- SG_ LKAS_ANGLE_CMD : 32|14@1- (0.1,0) [0|511] "" XXX
- SG_ LKAS_ANGLE_MAX_TORQUE : 55|8@0+ (1,0) [0|255] "" XXX
+BO_ 203 ADAS_CMD_35_10ms: 24 CGW_CCU
```

Comment:
This name change does make it more confusing to track how things work on the bus. Plus a comment that references `LFA_ALT` now doesn't have a message to reference

Reply:
I am agree, addressing it on https://github.com/commaai/opendbc/pull/2486/commits/1eec068bc141f22833a9af95f1de12477e983702